plugins/woocommerce/client/blocks/assets/js/blocks/product-elements/add-to-cart-form/edit.tsx

```diff
@@ -46,6 +47,15 @@ const AddToCartFormEdit = ( props: BlockEditProps< Attributes > ) => {
 		[]
 	);
 
+	// Check whether the current theme is block theme or not.
+	const isBlockTheme = useSelect( ( select ) => {
+		const themeSupports = select( coreStore ).getThemeSupports?.();
+		return themeSupports?.['block-templates'] === true;
+	}, [] );
```

Comment:
I don't have a strong opinion, but in order to keep it consistent. When we want to know if the site is running a block theme or not, what we do is save it from the PHP side of the block like here:

https://github.com/woocommerce/woocommerce/blob/e55e51d5461af425c86373af705626f3887cb1b5/plugins/woocommerce/src/Blocks/BlockTypes/ProductFilters.php#L38

And then you can simply access it from the JS side with `getSetting()`:

https://github.com/woocommerce/woocommerce/blob/e55e51d5461af425c86373af705626f3887cb1b5/plugins/woocommerce/client/blocks/assets/js/blocks/product-filters/frontend.ts#L193

How does it sound changing it? I don't have a strong preference towards one approach or the other, but I think it's better to keep it consistent accross blocks.